packages/shared/useTimeoutFn/index.ts

```diff
@@ -24,13 +25,17 @@ export function useTimeoutFn<CallbackFn extends AnyFn>(
   cb: CallbackFn,
   interval: MaybeRefOrGetter<number>,
   options: UseTimeoutFnOptions = {},
-): Stoppable<Parameters<CallbackFn> | []> {
+): Stoppable<Parameters<CallbackFn> | []> & Pausable & { timeLeft: ShallowRef<number> } {
   const {
     immediate = true,
   } = options
 
   const isPending = ref(false)
 
+  const isActive = ref(false)
+  const startedAt = ref(-1)
+  const timeLeft = ref(-1)
```

Comment:
I think this should not default to `-1` `new Date(-1)` is valid and `Thu Jan 01 1970 00:59:59`